### PR TITLE
Add `workflow` icon

### PR DESCRIPTION
**Pull request by danielbayley**
Like `workflow` from [octicons](https://primer.style/design/foundations/icons)…

### CODE REVIEW

File: icons/workflow.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <rect width="8" height="8" x="3" y="3" rx="2" />
+  <path d="M7 11v4c0 1.1.9 2 2 2h4" />
```
Comment (jguddas) [inline icons/workflow.svg:13]:
```suggestion
  <path d="M7 11v4a2 2 0 0 0 2 2h4" />
```

**Comment by jguddas** [comment]
Big fan of this one :+1:

**Review by ericfennis — APPROVED** [review]
Nice icons, clean and simple!

**Comment by karsa-mistmere** [comment]
LGTM, thanks @jguddas, I barely have to do any actual review. :D

**Pull request merged**